requirements.txt

```diff
@@ -0,0 +1,5 @@
+CardoExecutor==1.3.*
```

Comment:
next time dont mix 2 PR into one.
4/5 of the files you changed aren't requirements.txt related, yet you called the PR "Add requirements file"